### PR TITLE
Fix typos and markdowns

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell .`.

- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

### CODE REVIEW

File: HOWTO_RELEASE.md [hunk 1]
```diff
@@ -84,12 +80,11 @@ paste, but please pay attention to details!
     deactivate
     cd -
 
-
 ## Release
 
 - release in Github
 
-    xdg-open https://github.com/canonical/craft-cli/tags
+    xdg-open <https://github.com/canonical/craft-cli/tags>
```
Comment (facundobatista) [inline HOWTO_RELEASE.md:87]: This is wrong, it will make the command to redirect input/output instead of passing the argument.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-A Command Line Client builder that follows the [Canonical's Guidelines for a Command Line
-Interface](https://discourse.ubuntu.com/c/design/cli-guidelines/62).
+A Command Line Client builder that follows the [Canonical's Guidelines for a
+Command Line Interface](https://discourse.ubuntu.com/c/design/cli-guidelines/62).
 
-The library provides two main functionalities: 
+The library provides two main functionalities:
 
-- a framework to define and execute application commands, which involves argument parsing and the provision of help texts
+- a framework to define and execute application commands, which involves
```
Comment (facundobatista) [inline README.md:8]: Do not cut the lines, just let them flow. It's slightly worse when you see it not rendered, but you don't get the "lot of lines rewrapping diff mess" when changing the text.

File: HOWTO_RELEASE.md [hunk 3]
```diff
@@ -57,7 +54,6 @@ paste, but please pay attention to details!
     git tag -s X.Y.Z
     git push --tags
 
-
```
Comment (facundobatista) [inline HOWTO_RELEASE.md:60]: It's good to have two lines before titles, helps the eye separation. It's innocuous when rendered, too.

**Review by facundobatista — CHANGES_REQUESTED** [review]
Hello! Thanks for this PR! 

While it's fine for the real English typos, all other aesthetic changes should be reverted (I added inline comments). After that we can land it.

Thanks again!

**Review by facundobatista — APPROVED** [review]
Thanks!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**